harvest/time.go

```diff
@@ -0,0 +1,39 @@
+package harvest
+
+import (
+	"net/url"
+	"strings"
+	"time"
+)
+
+type Time struct {
+	time.Time
+}
+
+func (t Time) String() string {
+	return strings.ToLower(t.Time.Format(time.Kitchen))
+}
+
+func (t *Time) UnmarshalJSON(data []byte) (err error) {
+	str := string(data)
+	str = strings.Trim(str, "\"")
```

Comment:
REQ: It seems this isn't covering every case.
When doing some tests with your changes, I received `null` and `23:57` instead